pkg/webhook/namespacelabel.go

```diff
@@ -0,0 +1,85 @@
+package webhook
+
+import (
+	"context"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"net/http"
+
+	opa "github.com/open-policy-agent/frameworks/constraint/pkg/client"
+	"github.com/pkg/errors"
+	types "k8s.io/api/admission/v1beta1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+)
+
+var (
+	exemptNamespaces = newNSSet()
+)
+
+func init() {
+	AddToManagerFuncs = append(AddToManagerFuncs, AddLabelWebhook)
+	flag.Var(exemptNamespaces, "exempt-namespaces", "Namespaces that are allowed to set the admission.gatekeeper.sh/ignore label. To exempt multiple namespaces, this flag can be declared more than once.")
```

Comment:
should we call this `exempt-namespace`? since it exempts 1 namespace per line, and it doesn't parse comma separated multiple namespaces etc

Reply:
renamed and added the README changes discussed in the meeting this morning.